client/mds3/leftlabel.sample.js

```diff
@@ -29,6 +29,7 @@ export function makeSampleLabel(data, tk, block, laby) {
 			.attr('class', 'sja_clbtext2')
 			.style('opacity', 1)
 			.text(`${data.sampleTotalNumber} sample${data.sampleTotalNumber > 1 ? 's' : ''}`)
+			.attr('data-testid', 'sja-samples-label')
```

Comment:
please let me know if it's worth to "scope" the ids by app/component name prefix, e.g. all ids assigned to scripts in client/mds3/* are prefixed by `sjpp-mds3tk-`

Reply:
I've changed the label: `sja-samples-label` to `sjpp_mds3tk_samples_label`.
We need a clear standard on how to name `data-testids`. On the other hand, this is still a "proof of concept", and if it turns out that the approach is right and we continue to add e2e tests, then we can rename existing `data-testids`. There are just about 15 of them and it's easy to find them in the code. 
I would suggest we test this in prod first, to see how it works.

Reply:
sure thanks. also i prefer dash over underscore